projects/go-lib/src/lib/components/go-select/go-select.component.spec.ts

```diff
@@ -102,4 +102,19 @@ describe('GoSelectComponent', () => {
       expect(component.control.value).toEqual([1, 2, 3]);
     });
   });
+
+  describe('onRemoveAll()', () => {
+    it('uses bindValue to get value if bindValue exists', () => {
+      component.bindValue = 'id';
+      component.items = [
+        { id: 1, label: 'Label 1' },
+        { id: 2, label: 'Label 2' },
+        { id: 3, label: 'Label 3' }
+      ];
+
+      component.onRemoveAll();
```

Comment:
Should we set `component.control` to a value prior calling this method to ensure that the control actually goes from a value to null?

Reply:
Done this changes